File: HW/friend.cpp
#include <iostream>
#include <cstdio>
#include <cstring>
#include "friend.h"

using std::cin;
using std::cout;
using std::endl;
using std::string;

Friend::Friend(string &myfirstName, string &mylastName, int &myphoneNumber, int &myage) : Person(myfirstName, mylastName, myphoneNumber) 
{
    age = myage;
}

void Friend::setAge(int myage)
{
    age = myage;
}

int Friend::getAge()
{
    return age;
}

void Friend::print()
{
    cout << getFirstName() << " " << getLastName() << "_";
    int phoneNumber = getPhoneNumber();
    int_to_string(phoneNumber);
    cout << "_" << getAge() << endl;
}

File: HW_java/Friend.java
public class Friend extends Person
{
    private int age;

    public Friend(String myfirstName, String mylastName, int myphoneNumber, int myage)
    {
	super(myfirstName, mylastName, myphoneNumber);
	age = myage;
    }

    public void setAge(int myage)
    {
	age = myage;
    }

    public int getAge()
    {
	return age;
    }

    public void print()
    {
	System.out.print(getFirstName() + " " + getLastName() + "_");
	int phoneNumber = getPhoneNumber();
	int_to_string(phoneNumber);
	System.out.println("_" + getAge());	
    }
};
